python/src/equistore/labels.py

```diff
@@ -57,12 +57,19 @@ class Labels(np.ndarray):
             ...
     """
 
-    def __new__(cls, names: List[str], values: np.ndarray, **kwargs):
+    def __new__(cls, names: Union[List[str], str], values: np.ndarray, **kwargs):
         """
-        :param names: names of the variables in the new labels
+        :param names: names of the variables in the new labels, in the case of a single
+                      name also a single string can be given: ``names = ["name"]``
+                      is equivalent to ``names = "name"``
         :param values: values of the variables, this needs to be a 2D array of
             ``np.int32`` values
         """
+        if isinstance(names, str):
+            if len(names) == 0:
+                names = []
+            else:
+                names = [names]
```

Comment:
I would put this above L79 where we reformat names. Maybe one can even combine the logic you added with

```python
names = tuple(str(n) for n in names)
```

Reply:
do you like how it si now?

Reply:
Yes. Nicely solved!